index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/mobile_first.css">
+    <title>JD's Portfolio</title>
+</head>
+<body>
+    <div class="index-page">
+        <header>        
+        <nav class="flex-navbar">
+            <div>
+                <a href="#">
+                    <img class="primary-logo" src="img/archesportslogo.jpg" alt="ArchEsportsLogo">
+                </a>
+            </div>            
+            <ul class="flex-nav-list">
+               <li class="list-item"><a href="#">Portfolio</a></li>
+               <li class="list-item"><a href="#">About</a></li>
+               <li class="list-item"><a href="#"> Contact </a></li>
+            </ul>
+            <div class="menu">
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+                <div class="menu-line"></div>
+            </div>
+        </nav>
+        </header>
+        <section id="headline-section">
+            <h1 class="headline-title"> Hi there!👋🏼 <br>
+                I'm JD.</h1>
+                <h2 class="headline-sub-title"> I'm a Micronout 👨🏼‍🚀💻</h2>
+            <p>Hello, I'm Juan Diego. I'm a Microverse intern (Micronout) and I'm from Perú. <br>
+                I want to learn everything I need to be prepared for a programmer's job. You can contact me:
+            </p>
+               <footer>
+                <div id="contact">
+                    <ul>
+                        <li class="icon">
+                            <a href="https://github.com/JuanDiegoGuerra" rel="noopener" target="_blank">
+                                <img src="img/social media icons.png" alt="github link">
+                            </a></li>
+                    </ul>
+                </div>
```

Comment:
- [ ] Good job adding the `down arrow icon` as suggested by the previous reviewer. But please notice you need to make sure it's added to the right place on the webpage. currently, it's overlapping with your social media icons. This creates a bad experience for the end user, Please consider aligning it to the bottom and by its own. 

### Your design on `375px` screen size
![image](https://user-images.githubusercontent.com/69077061/227435251-6efe69ce-9b22-4183-b78a-724b44f5afba.png)